Kirstie-Whitaker/abstracts_bios_2020.md

```diff
@@ -0,0 +1,63 @@
+# Abstracts and bios
+
+This is a file of all the talk abstracts and bios I write for events in 2020.
+
+Hopefully I'll be able to re-use at least some of them ;)
+
+They're ordered by date of presentation so ideally they're a little easier to find.
+
+### Table of Contents
+
+* January
+* February
+* March
+* April
+* May
+* June
+* July
+* August
+* September
+* October
+* November
+* December
+
+### Manchester Literary and Philosophical Society
```

Comment:
I think it is really interesing talk.But maybe it should be showed in the table content.

Reply:
Added it! Thank you!